packages/core/src/utils/markers/index.ts

```diff
@@ -26,13 +27,28 @@ const makeMarker = ({
   fill = defaultFill,
 }: MarkerStyle) =>
   `${prefix}${encodeURIComponent(`
-<svg width="${imgSize[0]}" height="${imgSize[1]}" viewBox="0 0 30 43" xmlns="http://www.w3.org/2000/svg">
+<svg width="${imgSize[0]}" height="${
+    imgSize[1]
+  }" viewBox="0 0 30 43" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I can't seem to see the difference here on why the linter previously didn't add a linebreak.  
Nothing has changed, right? 🤨

Reply:
Yeah, that's weird. Maybe I changed something forth and back and this was the result that's *also* fine. I remember that I fiddled a little with height and width. But I'm not sure.